platform/openmote-cc2538/contiki-conf.h

```diff
@@ -79,6 +79,14 @@ typedef uint32_t rtimer_clock_t;
 #define RTIMER_CLOCK_DIFF(a, b)     ((int32_t)((a) - (b)))
 /** @} */
 /*---------------------------------------------------------------------------*/
+#define TSCH_CONF_HW_FRAME_FILTERING    0
```

Comment:
Do we want this set unconditionally? If needed only with TSCH let's add it to the list of platform-specific things we need to add for TSCH, see project-conf of rpl-tsch example. We need to make this configuration more smooth later but that is a different topic.